tests/js/cursor/range-3.js

```diff
@@ -1,5 +1,5 @@
-thisWontBeFormatted <|> ( 1  ,3<<<PRETTIER_RANGE_START>>>)
-
+thisWontBeFormatted <|> ( 1  ,3)
+<<<PRETTIER_RANGE_START>>>
     thisWillBeFormatted    (2  ,3,   )
 
  <<<PRETTIER_RANGE_END>>>   thisWontBeFormatted  (2, 90  ,)
```

Comment:
I believe this is a mistake, the `rangeStart` is inside first function call, the first line should be formatted.

Reply:
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEMAWBLAzgdWjAITgDEIAnAWwEMZ4ATAAgYAoGBGJgGgGYBKAHSiCmTdNhwYANpKKlKNeiJYAmLt04iBQqErG58s8tVpxGKjQE4ADF14hOICAAcYGaFmSgqZMhADuAAreCB4oVJJ+VACeHg4ARmRUYADWcDAAyk5JGFAA5sgwZACucA5wFHGmdKYAMlR5RVS5JEYKOflhRTAQ9iBoMBSSOJjwWFlgcOkhGK4AbjNRyOBYsSA5WHBkMAGJudTIAGbhGw4AVlgAHgSJKWnpVBRwNTlwh8elIOcX6e2ScACKRQg8DekhOICyZA2ZCWcSolUkvScZByMAkdHQyAAHFYHMiIBscIknEtkXBobNXg4AI5A+A7ZyhEBULAAWigcFMpl6ZDgtIwvJ2TX2SCOYI+GwoGAKxQlvwBdNeoveDhg8PRmKQylViSk7QAwhAKCKQOSAKy9IobAAq8NCYvBsxKAEkoNVYOkwCiXABBN3pGBRP6g8GJPKTNVbZDKLEOMPNACibuQAHZlABfdNAA)